topics/single-cell/tutorials/scrna-data-ingest/tutorial.md

```diff
@@ -0,0 +1,871 @@
+>    - *"using column"*: `c3` or `Column: 3`
+>    - In *"Check"*:
+>        - {% icon param-repeat %} *"Insert Check"*
+>            - *"Find Regex"*: `Symbol`
+>            - *"Replacement"*: `gene_short_name`
+> 2. Check that the datatype is `tabular`. If not, change it. 
+>
+>    {% snippet faqs/galaxy/datasets_change_datatype.md datatype="tabular" %}
+>     
+> 3. **Rename** {% icon galaxy-pencil %} the output: `Genes renamed`
+>
+{: .hands_on}
+
+We're almost there, but there is one last modification we have to do - transpose the matrix to have the genes as rows and cells as columns.
+
+> <hands-on-title> Transpose the matrix </hands-on-title>
+>
+> 1. {% tool [Transpose](toolshed.g2.bx.psu.edu/repos/iuc/datamash_transpose/datamash_transpose/1.8+galaxy1) %} with the following parameters:
+>    - {% icon param-file %} *"Input tabular dataset"*: `Expression matrix` 
+{: .hands_on}
+
+And the final step is to create the CDS file using Monocle tool!
+ 
+> <hands-on-title> Create Cell Data Set </hands-on-title>
+>
+> 1. {% tool [Monocle3 create](toolshed.g2.bx.psu.edu/repos/ebi-gxa/monocle3_create/monocle3_create/0.1.4+galaxy2) %} with the following parameters:
+>    - {% icon param-file %} *"Expression matrix, genes as rows, cells as columns. Required input. Provide as TSV, CSV or RDS."*: output of **Transpose** {% icon tool %}
+>    - *"Format of expression matrix"*: `TSV`
+>    - {% icon param-file %} *"Per-cell annotation, optional. Row names must match the column names of the expression matrix. Provide as TSV, CSV or RDS."*: `Cell barcodes (obs)` 
+>    - *"Format of cell metadata"*: `TSV`
+>    - {% icon param-file %} *"Per-gene annotation, optional. Row names must match the row names of the expression matrix. Provide as TSV, CSV or RDS."*: `Genes renamed` 
+>    - *"Format of gene annotation"*: `TSV`
+>
+> 2. **Rename** {% icon galaxy-pencil %} the output: `CDS Monocle file`
+{: .hands_on}
+
+As usual, you can check the [example history](https://usegalaxy.eu/u/j.jakiela/h/anndata-to-cell-data-set-cds-conversion) and the dedicated [workflow](https://usegalaxy.eu/u/j.jakiela/w/anndata-to-cell-data-set-cds-conversion) (it doesn't include the step on renaming the column header though). 
+
+</div>
+
+
+# Downsampling FASTQ files
+
+Sometimes, it is useful to work on smaller subsets of data (especially for teaching / learning purposes). Here is an example of how you can downsample your FASTQ files.
+
+First, let's get some toy data. We just need two FASTQ files - one containing barcodes, the other with transcripts. 
+
+> <hands-on-title>Get toy data</hands-on-title>
+>
+> 1. Create a new history for this section *"Downsampling FASTQ Files"*
+> 2. Import the files from [Zenodo]({{ page.zenodo_link }})
+>
+>      ```
+>      https://zenodo.org/record/4574153/files/SLX-7632.TAAGGCGA.N701.s_1.r_1.fq-400k.fastq
+>      https://zenodo.org/record/4574153/files/SLX-7632.TAAGGCGA.N701.s_1.r_2.fq-400k.fastq
+>      ```
```

Comment:
```suggestion
>    ```
>    https://zenodo.org/record/4574153/files/SLX-7632.TAAGGCGA.N701.s_1.r_1.fq-400k.fastq
>    https://zenodo.org/record/4574153/files/SLX-7632.TAAGGCGA.N701.s_1.r_2.fq-400k.fastq
>    ```
```

I suspect this will fix the linting error (I've seen it now twice on two PRs, where it has not once been triggered since we added that lint, which is, odd.)

Reply:
amazing, thanks for doing your magic <3